src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj

```diff
@@ -7,9 +7,7 @@
     <!-- CS0436 - System.Private.CoreLib has internals visible to System.Runtime.WindowsRuntime and is colliding internals on common shared code -->
     <NoWarn>$(NoWarn);1698;0436</NoWarn>
     <ProjectGuid>{844A2A0B-4169-49C3-B367-AFDC4894E487}</ProjectGuid>
-    <PackageTargetRuntime>win8</PackageTargetRuntime>
```

Comment:
Is there a reason you needed to remove these PackageTargetRuntime properties?

Reply:
IMHO we should.  We don't want to force folks to specify a windows version when publishing netcore apps.  We should make sure we test that the library doesn't blow-up if used on Win7.  I'd expect JIT to work and just some sort of natural runtime exception if you call into WinRT APIs.

Reply:
OK fine with removing them I was just curious what prompted it.